shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -1588,6 +1591,16 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(true)
 
+  val ENABLE_REWRITE_DATE_CONVERSION =
+    buildConf("spark.gluten.sql.rewrite.dateConversion")
+      .internal()
+      .doc(
+        "Rewrite the conversion between date and string."
```

Comment:
It is useful for velox backend?

Reply:
Not yet, beacuse this behavior may cause an inconsistent results. I have already change the config name to backend.ch.xx